lib/widgets/unread_count_badge.dart

```diff
@@ -65,3 +66,22 @@ class UnreadCountBadge extends StatelessWidget {
           count.toString())));
   }
 }
+
+class MutedUnreadBadge extends StatelessWidget {
+  const MutedUnreadBadge({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Opacity(
+      opacity: 0.5,
+      child: Container(
+        width: 8,
+        height: 8,
+        margin: const EdgeInsetsDirectional.only(end: 3),
+        decoration: const BoxDecoration(
+          color: HslColor(0, 0, 0.8),
+          shape: BoxShape.circle,),),);
```

Comment:
```suggestion
          color: HslColor(0, 0, 80),
          shape: BoxShape.circle)));
```
According to the web, `lightness = 80%` and in Flutter it's `80` too.
Also, those commas are not necessary if they're on the same line.

Reply:
Thanks! new revision pushed PTAL